server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/MailboxGetMethod.scala

```diff
@@ -43,34 +45,66 @@ class MailboxGetMethod @Inject() (serializer: Serializer,
                                   metricFactory: MetricFactory) extends Method {
   override val methodName: MethodName = MethodName("Mailbox/get")
 
-  override def process(invocation: Invocation, mailboxSession: MailboxSession): Publisher[Invocation] = {
+  object MailboxGetResults {
+    def found(mailbox: Mailbox): MailboxGetResults = MailboxGetResults(Set(mailbox), NotFound(Set.empty))
+    def notFound(mailboxId: MailboxId): MailboxGetResults = MailboxGetResults(Set.empty, NotFound(Set(mailboxId)))
+  }
+
+  case class MailboxGetResults(mailboxes: Set[Mailbox], notFound: NotFound) {
+    def merge(other: MailboxGetResults): MailboxGetResults = MailboxGetResults(this.mailboxes ++ other.mailboxes, this.notFound.merge(other.notFound))
```

Comment:
what is that mean? i just ask

Reply:
from the not review commit. But it is a method that merge two MailboxGetResults in one. 
By merging its found mailbox and notFound ids.